beginner_source/quickstart/build_model_tutorial.py

```diff
@@ -0,0 +1,153 @@
+"""
+Build Model Tutorial
+=======================================
+"""
+
+###############################################
+# The data has been loaded and transformed we can now build the model. 
+# We will leverage `torch.nn <https://pytorch.org/docs/stable/nn.html>`_ 
+# predefined layers that Pytorch has that can both simplify our code, and  make it faster.
+# 
+# In the below example, for our FashionMNIT image dataset, we are using a `Sequential` 
+# container from class `torch.nn. Sequential <https://pytorch.org/docs/stable/generated/torch.nn.Sequential.html>`_ 
+# that allows us to define the model layers inline. 
+# The neural network modules layers will be added to it in the order they are passed in.
+# 
+# Another way to bulid this model is with a class 
+# using `nn.Module <https://pytorch.org/docs/stable/generated/torch.nn.Module.html)>`_ This gives us more flexibility, because
+# we can construct layers of any complexity, including the ones with shared weights.
+#
+# Lets break down the steps to build this model below
+#
+
+##########################################
+# Inline nn.Sequential Example:
+# ----------------------------
+#
+
+import os
+import torch
+import torch.nn as nn
+import torch.onnx as onnx
+from torch.utils.data import DataLoader
+from torchvision import datasets, transforms
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+print('Using {} device'.format(device))
+
+# model
+model = nn.Sequential(
+        nn.Flatten(),
+        nn.Linear(28*28, 512),
+        nn.ReLU(),
+        nn.Linear(512, 512),
+        nn.ReLU(),
+        nn.Linear(512, len(classes)),
+        nn.Softmax(dim=1)
+    ).to(device)
+    
+print(model)
+
+#############################################
+# Class nn.Module Example:
+# --------------------------
+#
+
+class NeuralNework(nn.Module):
+    def __init__(self, x):
+        super(NeuralNework, self).__init__()
+        self.layer1 = nn.Linear(28*28, 512)
+        self.layer2 = nn.Linear(512, 512)
+        self.output = nn.Linear(512, 10)
+
+    def forward(self, x):
+        x = F.relu(self.layer1(x))
+        x = F.relu(self.layer2(x))
+        x = self.output(x)
+        return F.softmax(x, dim=1)
+
+#############################################
+# Get Device for Training
+# -----------------------
+# Here we check to see if `torch.cuda <https://pytorch.org/docs/stable/notes/cuda.html>`_ is available to use the GPU, else we will use the CPU. 
+#
+# Example:
+#
+
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+print('Using {} device'.format(device))
+
+##############################################
+# The Model Module Layers
```

Comment:
it would be helpful to have a snippet about what's happening in `__init__` (referencing `nn.Module`, initializing layers) before we dive into what each layer is